server/src/main/java/org/elasticsearch/repositories/blobstore/FileRestoreContext.java

```diff
@@ -118,7 +117,7 @@ public void restore(SnapshotFiles snapshotFiles) throws IOException {
                 // store can still have existing files but they will be deleted just before being
                 // restored.
                 recoveryTargetMetadata = indexShard.snapshotStoreMetadata();
-            } catch (org.apache.lucene.index.IndexNotFoundException e) {
+            } catch (IndexNotFoundException e) {
```

Comment:
I wanted to have this explicitly as a fully qualified name, in order to ensure that it does not fall victim to a refactoring to Elasticsearch's IndexNotFoundException, which recently happened.

Please revert this.

Reply:
reverted -> https://github.com/elastic/elasticsearch/pull/42193/commits/f2ab652fb1a49e8dc926436dcf4f79a9a176ffb2